gulp-tasks/build.js

```diff
@@ -47,6 +47,36 @@ const buildPackage = (projectPath) => {
     .then(() => printBuildTime(`${(Date.now() - startTime) / 1000}s`));
 };
 
+/**
+ * Updates the fields in package.json that contain version string to match
+ * the latest version from lerna.json.
+ *
+ * @param {String} projectPath The path to a project directory.
+ * @return {Promise} Resolves if updating succeeds, and rejects if it fails.
+ */
+const updateVersionedBundles = (projectPath) => {
```

Comment:
I thought packages only update when they've changed - this blanket updates everything to the latest version of packages - which may not be the case for some packages.

If this is running after lerna, then each package should be referencing it's own version.